consumer/ue_context_management.go

```diff
@@ -32,14 +33,23 @@ func UeCmRegistration(ue *amf_context.AmfUe, accessType models.AccessType, initi
 			InitialRegistrationInd: initialRegistrationInd,
 			Guami:                  &amfSelf.ServedGuamiList[0],
 			RatType:                ue.RatType,
-			// TODO: not support Homogenous Support of IMS Voice over PS Sessions this stage
-			ImsVoPs: models.ImsVoPs_HOMOGENEOUS_NON_SUPPORT,
+			ImsVoPs:                models.ImsVoPs_HOMOGENEOUS_NON_SUPPORT,
 		}
-		ctx, cancel := context.WithTimeout(context.TODO(), 30*time.Second)
+		gppAccessCtx, cancel := context.WithTimeout(ctx, 30*time.Second)
```

Comment:
Question: In all of the previous changes (other files), you "reuse" the `ctx` variable (e.g., `ctx, cancel := context.WithTimeout(ctx, 30*time.Second)` but here you are using a different name (i.e., `gppAccessCtx`). Any specific reason for this? Why not just naming it `ctx` as in the other files?

Reply:
The linter is complaining that the ctx variable declared with context.WithTimeout shadows the ctx parameter of the function when I do that. I'm not sure why it complains about it only here and not in other funcitons, it's likely because we're using it inside of a switch block.

Reply:
Ok